NorthstarDLL/squirrel/squirrel.cpp

```diff
@@ -209,7 +210,47 @@ template <ScriptContext context> void SquirrelManager<context>::VMCreated(CSquir
 
 template <ScriptContext context> void SquirrelManager<context>::VMDestroyed()
 {
+	// Call all registered mod Destroy callbacks.
+	if (g_pModManager)
+	{
+		NS::log::squirrel_logger<context>()->info("Calling Destroy callbacks for all loaded mods.");
+
+		for (const Mod& loadedMod : g_pModManager->m_LoadedMods)
+		{
+			for (const ModScript& script : loadedMod.Scripts)
+			{
+				for (const ModScriptCallback& callback : script.Callbacks)
+				{
+					if (callback.Context != context || callback.DestroyCallback.length() == 0)
+					{
+						continue;
+					}
+
+					SQObject functionobj {};
+					int result = sq_getfunction(m_pSQVM->sqvm, callback.DestroyCallback.c_str(), &functionobj, 0);
+					if (result != 0) // This func returns 0 on success for some reason
+					{
+						NS::log::squirrel_logger<context>()->error(
+							"VMDestroyed() was unable to find function with name '{}'. Is it global?", callback.DestroyCallback);
+						continue;
+					}
+
+					pushobject(m_pSQVM->sqvm, &functionobj);
+					pushroottable(m_pSQVM->sqvm);
+
+					_call(m_pSQVM->sqvm, 0);
```

Comment:
Why do you manually use `_call` here, as opposed to using `Call`?

Reply:
I just copied code from AsyncCall() tbh heh. I'll fix it.